distributed/tests/test_dask_collections.py

```diff
@@ -41,6 +43,9 @@ def assert_equal(a, b):
         assert a == b
 
 
+@pytest.mark.xfail(
+    dd._dask_expr_enabled(), reason="P2P transfer error", raises=RuntimeError
```

Comment:
This is https://github.com/dask-contrib/dask-expr/issues/955

Reply:
thanks! https://github.com/dask/distributed/pull/8560/commits/92204a5feb31324b65f83e8bfb8f582ee0abde20

Reply:
Actually, the issue is slightly different, I'll provide a fix.